lib/auth/webauthn/messages.go

```diff
@@ -37,6 +52,49 @@ type CredentialAssertionResponse struct {
 // registration.
 type CredentialCreation protocol.CredentialCreation
 
+func RequireResidentKey(cc *CredentialCreation) bool {
```

Comment:
Since we are attaching methods to the messages, why not attach this to CredentialCreation?